CSS/style.css

```diff
@@ -0,0 +1,188 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+body {
```

Comment:
[ OPTIONAL ] The page has a horizontal overflow which means some content is overflowing out of the container. Please see the screenshot below and fix the issue
![overflow](https://user-images.githubusercontent.com/35504149/134149526-a74ae2cb-972f-47de-85bc-30049205c92a.png)
.